doc/releasenotes/16_0_0_summary.md

```diff
@@ -280,6 +280,8 @@ is now fixed. The full issue can be found [here](https://github.com/vitessio/vit
 
 - vtbackup flag `--backup_storage_hook` has been removed, use one of the builtin compression algorithms or `--external-compressor` and `--external-decompressor` instead.
 
+- The VTTablet flag `--init_populate_metadata` has been deprecated, since we have deleted the `local_metadata` and `shard_metadata` sidecar database tables.
+
```

Comment:
Just to confirm:
on an upgrade of an existing vitess install that includes these tables, the new schema init will drop them. Correct?

Reply:
Not right now. We are not dropping tables that are on in specified list, to be safe and not result in lost data. But yes, the metadata tables should be deleted. Should I add explicit deletes for the metadata tables, or do you think we should drop any table not in the list?

Reply:
That's a good question, let us explicitly drop only metadata tables. Who knows what people might be storing in `_vt` that we don't know of.